plugins/extra/phrases.go

```diff
@@ -29,14 +40,65 @@ type phrase struct {
 	Deleted   bool
 }
 
-func newPhrasesPlugin(cm *seabird.CommandMux, db *nut.DB) error {
+func newPhrasesPlugin(b *seabird.Bot, cm *seabird.CommandMux, ndb *nut.DB, db *xorm.Engine) error {
+	l := b.GetLogger()
 	p := &phrasesPlugin{db: db}
 
-	err := p.db.EnsureBucket("phrases")
+	err := db.Sync(Phrase{})
 	if err != nil {
 		return err
 	}
 
+	rowCount, err := db.Count(Phrase{})
+	if err != nil {
+		return err
+	}
+
+	if ndb != nil && rowCount != 0 {
+		l.Info("Skipping phrases migration because target table is non-empty")
+	} else if ndb != nil {
```

Comment:
Would prefer this be split out so that the method is a little less complex

Reply:
Yeah, I'm a bit conflicted here. The current plan is just to tag a 1-time use version to migrate from nut to a real database and then completely remove nut (which would make these sections unneeded).

It'll give some time to move to a full migration system based on .Sync when we actually need it...

Reply:
This has been moved into a separate command for now